frontend/src/components/landing/BundleBanner.tsx

```diff
@@ -0,0 +1,134 @@
+import { Localized, useLocalization } from "@fluent/react";
+import { getBundlePrice } from "../../functions/getPlan";
+import { RuntimeData } from "../../hooks/api/runtimeData";
+import { Button } from "../Button";
+import { MozillaVpnWordmark } from "../Icons";
+import styles from "./BundleBanner.module.scss";
+import womanInBanner from "./images/bundle-banner-woman.png";
+import bundleFloatOne from "./images/bundle-float-1.svg";
+import bundleFloatTwo from "./images/bundle-float-2.svg";
+import bundleFloatThree from "./images/bundle-float-3.svg";
+import bundleLogo from "./images/vpn-and-relay-logo.svg";
+
+type FloatingFeaturesProps = {
+  icon: string;
+  text: string;
+  position: string;
+};
+
+export type Props = {
+  runtimeData: RuntimeData;
+};
+
+const FloatingFeatures = (props: FloatingFeaturesProps) => {
+  return (
+    <div
+      className={`${styles[props.position]} ${styles["float-features-item"]}`}
+    >
+      <img alt={props.text} src={props.icon} />
```

Comment:
This will result in `props.text` being read twice by screen readers: one for the image, and one for the caption. Since it's just a decorative image, we can silence it:

```suggestion
      <img alt="" src={props.icon} />
```

Reply:
Noted.